TeslaMateAgile/Services/BarryService.cs

```diff
@@ -23,7 +23,7 @@ public async Task<IEnumerable<Price>> GetPriceData(DateTimeOffset from, DateTime
     {
         var request = new BarryRequest
         {
-            Params = new string[] { _options.MPID, from.UtcDateTime.ToString("o"), to.UtcDateTime.ToString("o") }.ToList()
+            Params = new string[] { _options.MPID, from.UtcDateTime.ToString("s") + "Z", to.UtcDateTime.ToString("s") + "Z" }.ToList()
```

Comment:
Can you explain the reasoning behind this change and why it was necessary?

Reply:
Barry should just be ignored (probably deleted?)